contrib/zmq/zmq_sub.py

```diff
@@ -34,7 +34,7 @@
     print("This example only works with Python 3.5 and greater")
     sys.exit(1)
 
-port = 28332
+port = 48332
```

Comment:
The [wiki](https://github.com/bitcoin/bips/blob/master/bip-0094.mediawiki) says the default port is `48333`.  Is this a typo?

Reply:
48333 is the default p2p port

48332 is used for RPC, which is not defined in the BIP because it's specific to Bitcoin Core (as is ZMQ).